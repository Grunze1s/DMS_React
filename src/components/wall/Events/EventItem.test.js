import React from 'react';
import EventItem from './EventItem';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < EventItem {...props}  />)
    )
}

const dummy_data={
  data:{  image:"image.png",
    title:"hello",
    date:"20 Nov, 2020",
    address:"kupondole"
}
}
describe('<TestComponent />', () => {
it("EventItem render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

