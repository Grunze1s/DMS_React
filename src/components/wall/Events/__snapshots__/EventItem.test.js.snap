// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestComponent /> EventItem render 1`] = `ShallowWrapper {}`;
